src/scenarios/helpers/assetPacks.ts

```diff
@@ -24,10 +24,13 @@ export function makeFakeAssetPack(assetCount: number = 1): FullAssetPack {
   const id = uuid.v4()
   const categories = ['nature', 'furniture', 'items', 'misc']
 
-  let out: { id: string; version: number; title: string; assets: Asset[] } = {
+  let out: FullAssetPack = {
     id,
-    version: 1,
     title: 'Default Asset Pack',
+    thumbnail: '',
+    url: '',
+    isNew: false,
```

Comment:
We should have another field in the ui.sidebar domain where new packs are stored (only runtime tho)